README.md

```diff
@@ -11,14 +11,14 @@ A lightweight library to decode and verify RS JWT meant for the browser.
 ## Usage
 
 ```js
-var IdTokenVerifier = require('idtoken-verifier');
+import IdTokenVerifier from 'idtoken-verifier';
```

Comment:
Wonder if we should leave this as var and function syntax for compatibility with older browser at least in the README just in case they're not using babel etc.

Reply:
`import/export` is a valid syntax today for all major browsers. `var/require` isn't. So, if they're using var/require, they're using babel or something like that for sure.

![image](https://user-images.githubusercontent.com/941075/53198189-8884a000-35fa-11e9-9b9b-7638839b2772.png)